cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/DefaultScheduleBuilder.java

```diff
@@ -118,22 +112,22 @@ public ConstraintProgramScheduleBuilder withDurationSinceLastRun(long duration,
 
   @Override
   public ScheduleCreationSpec triggerByTime(String cronExpression) {
-    return new ScheduleCreationSpec(name, description, programName, properties, new TimeTrigger(cronExpression),
```

Comment:
Wouldn't it be cleaner if each of these `triggerByTime`, `triggerOnPartitions`, called `triggerOn`?

It would certainly save the redundancy of having `return new ScheduleCreationSpec(...)` for each of these methods, and `triggerOn` already accounts for if the `Trigger` is a `TriggerBuilder`, so it's has safe typecasting.